cg/services/order_validation_service/models/order_with_cases.py

```diff
@@ -32,3 +33,10 @@ def enumerated_existing_cases(self) -> list[tuple[int, Case]]:
             if not case.is_new:
                 cases.append((case_index, case))
         return cases
+
+    @property
+    def samples(self) -> list[Sample]:
+        samples = []
+        for case in self.cases:
+            samples.extend(case.samples)
```

Comment:
We will have some troubles with existing cases since they do not have a .samples property. We probably need something more involved

Reply:
Removed the usage in the tickethandler and used the new property